### ISSUE TITLE
Please rethink your CCT structure

### ISSUE THREAD

**Issue by jcc5018** [CrocoBlock/suggestions#7171]
You all have a CCT which is basically a CPT with organized data. This is a very welcomed feature, but the way you all have it implemented is a bit too complicated. 

Most people have CPT's already. Having to convert to a CCT just to have data organized in a dedicated table is a pain in the butt. Especially if your process involves first creating a CCT then connecting it to a CPT and having to map fields for title or body text. 

What about existing Cpt's where I only want my data out of the post_meta table. 

It would be a lot simpler to add a "Store meta fields in dedicated table" to the CPT post creation screen then having to deal with multiple different listings CCT/ CPT

Most cases of what you call CCT's are going to be from people who want to create a custom post type with lots of extra fields. I cant think of any use case where someone would not want a single post template of these items, and limiting it, is just alienating a lot of potential customers. 

It is a very simple solution. Everything (description and title) is already added in the posts table.    This is assuming you arent able to make everything in its own table and loop similar to Posts which would be ideal. 

If a dedicated table is selected, all you need is a new table with a post_id foreign key.  Most everything else should use the same code as your CCT's but without the extra confusion for your users. 

![image](https://github.com/Crocoblock/suggestions/assets/21000134/947d6172-e1bc-4016-8803-9f4980a17df7)

Having these options in CCT  should be automatic. 

![image](https://github.com/Crocoblock/suggestions/assets/21000134/771d2e73-eb17-4798-8581-3ef34ea467cc)

In summary your CCT concept is right, but the way you have it implemented is overly complicated and confusing. And because it would involve me completely redoing my preexisting CPT's, It may end up being a deal breaker for updating my license.  

If you would like additional feedback on this issue, let me know.

**Comment by MjHead** [comment]
Separate DB table for Post Meta is already in the roadmap for the future releases. And similar issues was already created - https://github.com/Crocoblock/suggestions/issues/6015 

At the moment we not 100% sure - will be this feature based on CCT or not - cant say at the moment. We'll make this decision after the research before starting to work on this
